src/tools/dartpad/index.md

```diff
@@ -82,14 +77,13 @@ To create a simple command-line app, use **New Pad**.
   Click the **New Pad** button,
   and confirm that you want to discard changes to the current pad.
 
-  The source code for the Hello World app appears
-  under the DART tab.
+  Click on the Dart logo to access the source code for
+  the Hello World app.
   </li>
 
   <li markdown="1">
-  Turn off **HTML**, using the toggle
-  underneath the Dart logo.
-  The HTML and CSS tabs disappear.
+  Using the toggle underneath the logo,
+  verify **HTML** support is disabled.
```

Comment:
```suggestion
  Click the Dart logo, make sure that **HTML** support is disabled,
  and then click **Create**.
```


Btw, we don't say "click _on_" (https://developers.google.com/style/word-list#letter-c), and we try to be more specific than _access_ (https://developers.google.com/style/word-list#letter-a).